_data/harnesses/versioned-ktables/kstreams.yml

```diff
@@ -0,0 +1,125 @@
+        file: tutorials/versioned-ktables/kstreams/markup/dev/make-docker-compose.adoc
+    - action: execute_async
+      file: tutorial-steps/dev/docker-compose-up.sh
+      render:
+        file: tutorials/versioned-ktables/kstreams/markup/dev/start-compose.adoc
+    - action: execute
+      file: tutorial-steps/dev/wait-for-containers.sh
+      render:
+        skip: true
+  - title: Configure the project
+    content:
+    - action: make_file
+      file: build.gradle
+      render:
+        file: tutorials/versioned-ktables/kstreams/markup/dev/make-build-file.adoc
+    - action: execute
+      file: tutorial-steps/dev/gradle-wrapper.sh
+      render:
+        file: tutorials/versioned-ktables/kstreams/markup/dev/make-gradle-wrapper.adoc
+    - action: execute
+      file: tutorial-steps/dev/make-configuration-dir.sh
+      render:
+        file: tutorials/versioned-ktables/kstreams/markup/dev/make-config-dir.adoc
+    - action: make_file
+      file: configuration/dev.properties
+      render:
+        file: tutorials/versioned-ktables/kstreams/markup/dev/make-dev-file.adoc
+  - title: Create the Kafka Streams topology
+    content:
+    - action: execute
+      file: tutorial-steps/dev/make-src-dir.sh
+      render:
+        file: tutorials/versioned-ktables/kstreams/markup/dev/make-src-dir.adoc
+    - action: make_file
+      file: src/main/java/io/confluent/developer/VersionedKTableExample.java
+      render:
+        file: tutorials/versioned-ktables/kstreams/markup/dev/make-topology.adoc
+  - title: Compile and run the Kafka Streams program
+    content:
+    - action: execute
+      file: tutorial-steps/dev/build-uberjar.sh
+      render:
+        file: tutorials/versioned-ktables/kstreams/markup/dev/build-uberjar.adoc
+    - action: execute_async
+      file: tutorial-steps/dev/run-dev-app.sh
+      render:
+        file: tutorials/versioned-ktables/kstreams/markup/dev/run-dev-app.adoc
+    - name: wait for streams app to be fully running
+      action: sleep
+      ms: 10000
+      render:
+        skip: true
+  - title: Consume data from the output topic
+    content:
+    - action: execute_async
+      file: tutorial-steps/dev/harness-console-consumer.sh
+      stdout: tutorial-steps/dev/outputs/actual-output.txt
+      render:
+        file: tutorials/versioned-ktables/kstreams/markup/dev/run-consumer.adoc
+    - action: execute_async
+      file: tutorial-steps/dev/harness-console-consumer-verify-table.sh
+      stdout: tutorial-steps/dev/outputs/actual-output-validate-table.txt
+      render:
+        skip: true
+    - name: wait for the consumer to read the messages
+      action: sleep
+      ms: 15000
+      render:
+        skip: true
+test:
+  steps:
+  - title: Create a test configuration file
+    content:
+    - action: make_file
+      file: configuration/test.properties
+      render:
+        file: tutorials/versioned-ktables/kstreams/markup/test/make-test-file.adoc
+  - title: Test the Versioned KTable Topology
```

Comment:
Capitalization nit - I _think_ that this be "Test the versioned KTable topology" since versioned ktable isn't proper noun, right? and same with topology

Reply:
ack